frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -341,8 +346,20 @@ function ReaderFooter:updateFooterContainer()
         }
     end
 
-    table.insert(self.vertical_frame, self.footer_container)
+    local vertical_span = VerticalSpan:new{width = self.bottom_padding *2}
 
+    if self.settings.progress_bar_position == "above" and not self.settings.disable_progress_bar then
+        table.insert(self.vertical_frame, self.progress_bar)
+        table.insert(self.vertical_frame, vertical_span)
+        table.insert(self.vertical_frame, self.footer_container)
+    elseif self.settings.progress_bar_position == "below" and not self.settings.disable_progress_bar then
+        table.insert(self.vertical_frame, self.footer_container)
+        table.insert(self.vertical_frame, vertical_span)
+        table.insert(self.vertical_frame, self.progress_bar)
+    else
+        table.insert(self.vertical_frame, vertical_span)
```

Comment:
Could it be because of this vertical_span? which wasn't added before?

Reply:
That might track with the extra blank pixel line @Frenzie originally mentioned (one on top, one on the bottom)?

Reply:
No pb for me with the space below the text.
Which might not be any additional space: note the bar/pipe separator character in my screenshots above, that goes touching the bottom of the screen. So, that space is just the normal space below the baseline for our footer font. Don't reduce it ! :)

Reply:
True. Plus, that was with the "thin" bar, not the classic one ;).

Reply:
Yes, I added this line (extra vertical space). Restore as it was before?

Reply:
Yes, I'd prefer it as before, as currently you add a non-cancellable bottom margin (so, when you set the bottom margin to 0, you wouldn't get 0 in the end) - and one can always add it if he wants it by using a bigger bottom margin.

Reply:
Ok, I will do :)